### PR TITLE
lookups

### PR DESCRIPTION
**Pull request by aileenmcd**
Adding some of the lookups between lower and higher geographies for given years (including 2 new lookups, LAD to Combined Authorities and LAD to Fire & Rescue Authorities). 
Followed the column naming convention on the current tables in geographr e.g. lad_19_name etc.

### CODE REVIEW

File: data-raw/query-urls.R [hunk 1]
```diff
@@ -33,12 +33,23 @@ query_urls <-
...
+    "lookup", "lsoa_msoa_11", "03.08.18", "OGLv3", "https://opendata.arcgis.com/datasets/fe6c55f0924b4734adf1cf7104a0173e_0.csv", "http://geoportal.statistics.gov.uk/datasets/fe6c55f0924b4734adf1cf7104a0173e_0",
+    "lookup", "postcode_oa_lsoa_msoa_lad_20", "23.11.20", "OGLv3", "https://www.arcgis.com/sharing/rest/content/items/9f629cd0c2fb42beb0d79a474fee4776/data", "https://geoportal.statistics.gov.uk/datasets/9f629cd0c2fb42beb0d79a474fee4776/about",
+    "lookup", "lad_cauth_20", "27.11.2020","OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LAD20_CAUTH20_EN_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/ons::local-authority-district-to-combined-authority-december-2020-lookup-in-england/about",
+    "lookup", "lad_fra_21", "30.03.21","OGLv3", "https://services1.arcgis.com/ESMARspQHYMw9BZ9/arcgis/rest/services/LAD21_FRA21_EW_LU/FeatureServer/0/query?where=1%3D1&outFields=*&outSR=4326&f=json", "https://geoportal.statistics.gov.uk/datasets/ons::local-authority-district-to-fire-and-rescue-authority-april-2021-lookup-in-england-and-wales/about",
```
Comment (MikeJohnPage) [inline data-raw/query-urls.R:46]: Please rearrange lookups into alphabetical order for easier maintenance/reference

File: data-raw/lookup_msoa_lad_19.R [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  pull(query)
+
+msoa_lad <-
+  read_sf(query_url)
+
+# Select and rename vars
+msoa_lad <-
+  msoa_lad |>
+  select(
+    msoa_11_name = MSOA11NM,
```
Comment (MikeJohnPage) [inline data-raw/lookup_msoa_lad_19.R:22]: I think the fact that some lookup tables use different year codes creates a problem. It isn't clear from just the name of the variable alone, what year the lookups cover. For example for this variable `lookup_msoa_lad_19`, I would expect both the msoa and lad codes to be 2019.

I think a better naming system would be to use `lookup_geographr1_year1_geography2_year2`. In this instance that would mean renaming the file, variable, query-urls.R table, and data.R documentation to use lookup_msoa_11_lad_19.

Do you agree with this new system? If so please can you update the lookups to match this new schema? The lookup `lookup_lad_lad` can be left as is, as it covers many years.

File: data-raw/query-urls.R (hunk 1 above)
Comment (aileenmcd) [reply data-raw/query-urls.R]: Resolved in https://github.com/humaniverse/geographr/pull/24/commits/e1ceb0d88b1da644f9be739f026c2d73123fd31b.

File: data-raw/lookup_msoa_lad_19.R (hunk 2 above)
Comment (aileenmcd) [reply data-raw/lookup_msoa_lad_19.R]: Yes good idea, particularly with the new census data coming. Have made the changing to names in commit https://github.com/humaniverse/geographr/pull/24/commits/a6f1fca2b0e9106ad83d0ac04998ab554e1d289f.

Comment (aileenmcd) [reply data-raw/lookup_msoa_lad_19.R]: Also I will update demograhr so it has the same naming convention for column names making the year codes explicit e.g. `lad_21_name` vs. `lad_name` to make joining between tables in geographr and demographr.

Comment (MikeJohnPage) [reply data-raw/lookup_msoa_lad_19.R]: Thanks for the updates

**Pull request merged**